### PR TITLE
Added unit tests

### PR DESCRIPTION
**Pull request by mickvav**
## Summary by CodeRabbit

- **New Features**
	- Added a string representation method for the LinkedList class, improving user-friendly output when printing the list.

- **Tests**
	- Introduced a suite of unit tests for the LinkedList class, enhancing testing coverage for adding elements.
	- Added methods to verify the functionality of adding elements at the head, end, and second position in the list.

### CODE REVIEW

**Review by TatianaErshova — APPROVED** [review]
LGTM

**Review by asershova — APPROVED** [review]
LGTM

File: data_structures/linked_list/linked_list_test.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+class TestLinkedList(unittest.TestCase):
+    
+        def test_add(self):
+            ll = LinkedList()
+            ll.add(1)
+            self.assertEqual(ll.head.data, 1)
+            ll.add(2)
+            self.assertEqual(ll.head.data, 2)
+
```
Comment (mickvav) [inline data_structures/linked_list/linked_list_test.py:14]: something

File: data_structures/linked_list/linked_list_test.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            self.assertEqual(ll.head.next.data, 2)
+    
+        def test_addtosecond(self):
+            ll = LinkedList()
+            ll.add(1)
+            ll.add(2) # 2 -> 1
+            ll.addtosecond(3) # 2 -> 3 -> 1
+            self.assertEqual(ll.head.data, 2)
+            self.assertEqual(ll.head.next.data, 3)
+            self.assertEqual(ll.head.next.next.data, 1)
```
Comment (mickvav) [inline data_structures/linked_list/linked_list_test.py:30]:
```suggestion
        def test_addtosecond(self):
            ll = LinkedList()
            ll.add(1) # 1
            ll.add(2) # 2 -> 1
            ll.addtosecond(3) # 2 -> 3 -> 1
            self.assertEqual(ll.head.data, 2)
            self.assertEqual(ll.head.next.data, 3)
            self.assertEqual(ll.head.next.next.data, 1)
```

**Pull request merged**